content/es/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,164 @@
+sistemas de archivos, no ven los procesos de los demás y el uso de recursos
+puede ser limitado. Son más fáciles de construir que una máquina virtual, y
+porque no están acoplados a la infraestructura y sistema de archivos del
+anfitrión, pueden llevarse entre nubes y distribuciones de sistema operativo.
+
+Ya que los contenedores son pequeños y rápidos, una aplicación puede ser
+empaquetada en una imagen de contenedor. Esta relación uno a uno entre
+aplicación e imagen nos abre un abanico de beneficios para usar contenedores.
+Con contenedores, podemos crear imágenes inmutables al momento de la compilación
+en vez del despliegue ya que las aplicaciones no necesitan componerse junto al
+resto de la pila ni atarse al ambiente de infraestructura de producción. Generar
+una imagen de contenedor al momento de la compilación permite tener un ambiente
+consistente que va desde desarrollo hasta producción. De igual forma, los contenedores
+son más transparentes que las máquinas virtuales y eso hace que el monitoreo y la
+administración sean más fáciles. Esto se aprecia más cuando los ciclos de vida de
+los contenedores son administrados por la infraestructura en vez de un proceso supervisor
+escondido en el contenedor. Por último, ya que solo hay una aplicación por contenedor,
+administrar el despliegue de la aplicación se reduce a administrar el contenedor.
+
+En resumen, los beneficios de usar contenedores incluyen:
+
+* **Ágil creación y despliegue de aplicaciones**:
+    Mayor facilidad y eficiencia al crear imágenes de contenedor en vez de máquinas virtuales
+* **Desarrollo, integración y despliegue continuos**:
+    Permite que la imagen de contenedor se construya y despliegue de forma frecuente y confiable,
+    facilitando los rollbacks pues la imagen es inmutable
+* **Separación de tareas entre Dev y Ops**:
+    Puedes crear imágenes de contenedor al momento de compilar y no al desplegar, desacoplando la
+    aplicación de la infraestructura
+* **Observabilidad**
+    No solamente se presenta la información y métricas del sistema operativo, sino la salud de la
+    aplicación y otras señales
+* **Consistencia entre los ambientes de desarrollo, pruebas y producción**:
+    La aplicación funciona igual en un laptop y en la nube
+* **Portabilidad entre nubes y distribuciones**:
+    Funciona en Ubuntu, RHEL, CoreOS, tu datacenter físico, Google Kubernetes Engine y todo lo demás
+* **Administración centrada en la aplicación**:
+    Eleva el nivel de abstracción del sistema operativo y el hardware virtualizado a la aplicación que funciona en un sistema con recursos lógicos
+* **[Microservicios](https://martinfowler.com/articles/microservices.html)** distribuidos, elásticos, liberados y débilmente acoplados:
+    Las aplicaciones se separan en piezas pequeñas e independientes que pueden ser desplegadas y administradas de forma dinámica, y no como una pila monolítica que opera en una sola máquina grande
```

Comment:
```suggestion
    Las aplicaciones se separan en piezas pequeñas e independientes que pueden ser desplegadas y administradas de forma dinámica, y no como una aplicación monolítica que opera en una sola máquina de gran capacidad
```

Reply:
Igual que antes, creo que pila no se suele usar mucho en este contexto. Y la otra,`máquina grande` suena un poco brusco. Up tou you ambas suggerencias!

Reply:
Si que me consigo pila con bastante frecuencia, pero siempre soy partidario de dejar en inglés (en itálicas) un término de uso común como stack. En este caso, creo que _stack_ no era la mejor palabra en inglés y aplicación está bien.